lib/impl/AzurePresignTvm.js

```diff
@@ -43,9 +44,16 @@ class AzurePresignTvm extends Tvm {
    * @private
    */
   async _generateCredentials (params) {
+    const accountURL = `https://${params.azureStorageAccount}.blob.core.windows.net`
     const sharedKeyCredential = new azure.SharedKeyCredential(params.azureStorageAccount, params.azureStorageAccessKey)
     const containerName = Tvm._hash(params.owNamespace)
 
+    const pipeline = azure.StorageURL.newPipeline(sharedKeyCredential)
+    const serviceURL = new azure.ServiceURL(accountURL, pipeline)
+    const privateContainerURL = azure.ContainerURL.fromServiceURL(serviceURL, containerName)
+
+    const identifier = azureUtil.getAccessPolicy(privateContainerURL, params.azureStorageAccount, params.azureStorageAccessKey)
```

Comment:
what if there is no policy yet ? I know it's unlikely as users should call the AzureBlobTvm first, but maybe we should throw an error here ?